src/sql/workbench/parts/notebook/notebookStyles.ts

```diff
@@ -231,5 +235,23 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean): IDi
 			}
 			`);
 		}
+
+		// Results grid options. Putting these here since query editor only adds them on query editor load.
```

Comment:
@anthonydresser please give feedback on this. I think we could just leave the logic here - it will be updated on configuration change like in the editors.